src/extensions/service-managers/civicrm-donotsms/index.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-empty-function */
+/* eslint-disable no-unused-vars */
+// This is the civicrm-donotsms service manager.
+
+import { optoutContactToGroup } from "../../contact-loaders/civicrm/util";
+// import { log } from "../../../lib/log";
+
+export const name = "civicrm-donotsms";
+
+export const metadata = () => ({
+  displayName: "The 'Do Not SMS' Opt out service manager",
+  description: "Used to opt folk out of receiving SMSes",
+  canSpendMoney: false,
+  moneySpendingOperations: [],
+  supportsOrgConfig: true,
+  supportsCampaignConfig: true
+});
+
+// When a contact is opted out
+// ! This is not currently functional, see https://github.com/MoveOnOrg/Spoke/issues/2069
```

Comment:
This looks like it was fixed in 12.0, so I'd delete this comment to avoid confusion